### PR TITLE
feature/about-markup

### PR DESCRIPTION
**Pull request by valeriya2022v**
Add about-markup

### CODE REVIEW

**Comment by Marrkelov** [comment]
Правки

**Comment by valeriya2022v** [comment]
Правки

File: src/partials/about.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<section class="about">
+    <div class="container">
+        <p class="title-label--white">tradition and love</p>
```
[the remark this answers is not in the document]
Comment (Marrkelov) [reply src/partials/about.html:3]: about__overtitle

File: src/partials/about.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            we follow it when creating our handmade desserts.
+        </p>
+        <p class="about__text">
+            We use traditional recipes and 100% natural ingredients. Our products are so natural that they can be given to children
+            from the age of three years. Sweets lovers can expect 55 types of ice cream, 15 types of coffee and 23 milkshakes.
+        </p>
+        <p class="about__text">
+            This is enough to choose ice cream and coffee for any mood!
+        </p>
+        <button class="about-button" type="button">
```
[the remark this answers is not in the document]
Comment (Marrkelov) [reply src/partials/about.html:40]: about__button

**Pull request merged**